index.js

```diff
@@ -12,3 +15,27 @@ const pagination = document.querySelector('[data-js="pagination"]');
 const maxPage = 1;
 const page = 1;
 const searchQuery = "";
+
+// Fetching Data
+async function fetchCharacters(slug) {
+  cardContainer.innerHTML = "";
+  const url = `https://rickandmortyapi.com/api/${slug}`;
```

Comment:
You do not really need to save the url in a `const` here, you can just pass the url directly as an argument to the `fetch()`.